### ISSUE TITLE
biallelic_autosomal

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7494]
**Preferred term label:**
biallelic_autosomal

**Synonyms**
Autosomal Recessive 
HP:0000007

**Definition (free text, please give PubMed ID)**
The allelic requirement for autosomal recessive inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
